sormas-ui/src/main/java/de/symeda/sormas/ui/externalmessage/processing/AbstractProcessingFlow.java

```diff
@@ -127,6 +131,24 @@ protected CaseDataDto buildCase(PersonDto person, ExternalMessageDto externalMes
 		caseDto.setReportingUser(user.toReference());
 		caseDto.setReportDate(
 			externalMessageDto.getCaseReportDate() != null ? externalMessageDto.getCaseReportDate() : externalMessageDto.getMessageDateTime());
+
+		FacilityReferenceDto personFacility = externalMessageDto.getPersonFacility();
+		if (personFacility != null && FacadeProvider.getConfigFacade().isConfiguredCountry(CountryHelper.COUNTRY_CODE_LUXEMBOURG)) {
```

Comment:
I'd like to make this not exclusive to Luxembourg because it should be a general feature of the processing flow; will this break DEMIS if you remove the country check?

Reply:
I doesn't break anything so I removed the server country check